tests/test_qdrant_client.py

```diff
@@ -62,44 +64,66 @@ def create_random_vectors():
     return vectors_path
 
 
-def test_rest_init():
+def test_client_init():
+    import tempfile
```

Comment:
why is it here

Reply:
to test `path=/tmp/...`

Reply:
I mean why the import is inside the function, any particular reason?

Reply:
it's just a nit comment